src/environments/environment.prod.ts

```diff
@@ -1,5 +1,3 @@
-/** @ignore */
 export const environment = {
    production: true,
-   cms_url: 'http://cms-dev-1861735057.us-east-1.elb.amazonaws.com:9009/CMSforce/as'
```

Comment:
make sure you're not changing environment info

Reply:
I'll take care of it.